spring-cloud-netflix-eureka-client/src/main/java/org/springframework/cloud/netflix/eureka/EurekaClientConfigBean.java

```diff
@@ -413,6 +413,13 @@
 	 */
 	private int order = 0;
 
+	/**
+	 * A temporary property to switch between using either RestTemplate or WebClient
+	 * support.
+	 * @return
+	 */
+	private boolean webclientSupport = false;
```

Comment:
let's make the property `eureka.client.webclient.enabled` and move the definition to https://github.com/spring-cloud/spring-cloud-netflix/blob/master/spring-cloud-netflix-eureka-client/src/main/resources/META-INF/additional-spring-configuration-metadata.json rather than here in the config bean.

Reply:
Done.